checksec.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eu
+
+require_from() {
+    if ! command -v "$1" >/dev/null 2>/dev/null ; then
+        echo "Please acquire $1 from $2" >&2
+        exit 1
+    fi
+}
+
+require_from checksec    https://github.com/slimm609/checksec.sh
+require_from jq          https://stedolan.github.io/jq/
```

Comment:
AFAIK `checksec` also requires a `readelf` executable...should we also validate that it exists and prompt the user to install `binutils` if it doesn't?